finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
Exactly! Although, I don't understand why we'd need to touch ``ServerBridge`. It's a ChannelInitialize right now that [essentially an InboundHanlder](https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/ChannelInitializer.java#L53) that will receive `channelRegistered` (or `channelInit` in the `ChannelInitializer` case) as any other handler.